ibm-messaging-mq-cloud-showcase-app/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This is a duplicate of the LICENSE at the root of dev-patterns. This one can be removed.